src/webgl/p5.Framebuffer.js

```diff
@@ -1479,6 +1479,35 @@ class Framebuffer {
    * @param  {Number} w width of the subsection to be returned.
    * @param  {Number} h height of the subsection to be returned.
    * @return {p5.Image} subsection as a <a href="#/p5.Image">p5.Image</a> object.
+   *
+   * @example
+   * <div>
+   * <code>
+   * // The `pixelColor` is logged to the console, returning red with full opacity.
+   *
+   * function setup() {
+   *   createCanvas(400, 400, WEBGL);
```

Comment:
Oh sorry one last thing, 400x400 will be rather large for the reference, where examples are typically use 100x100 for pages like https://p5js.org/reference/p5/createFilterShader/. Could we maybe do the same here? There's also nothing wrong with a framebuffer larger than the main canvas, but it may be helpful for illustration purposes to also make `myBuffer` fit in that size too.

Reply:
No worries